src/System.Management.Automation/engine/NativeCommandParameterBinder.cs

```diff
@@ -349,6 +352,35 @@ internal static bool NeedQuotes(string stringToCheck)
             return needQuotes;
         }
 
+        static private string GetEnumerableArgSeparator(ArrayLiteralAst arrayLiteralAst, int index)
+        {
+            if (arrayLiteralAst == null) return " ";
+
+            // index points to the *next* element, so we're looking for space between
+            // it and the previous element.
+
+            var next = arrayLiteralAst.Elements[index];
+            var prev = arrayLiteralAst.Elements[index - 1];
+
+            var arrayExtent = arrayLiteralAst.Extent;
+            var afterPrev = prev.Extent.EndOffset;
+            var beforeNext = next.Extent.StartOffset - 1;
+
+            if (afterPrev == beforeNext)
+                return ",";
```

Comment:
Could you please use one line formatting as above in line 357?
Below too in 376 and 379.